src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -51,33 +62,36 @@ enum AddressSuggestionRadioButton {
 }
 
 const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
-  verifiedAddressResult,
+  verifyAddress,
```

Comment:
This was one of the changes in https://github.com/artsy/force/pull/12636 which i thought were worthwhile.